accounts-db/src/accounts_db.rs

```diff
@@ -15908,6 +16136,8 @@ pub mod tests {
                                 debug!("space: {space}, lamports: {lamports}, alive: {alive}, account_count: {account_count}, append_opposite_alive_account: {append_opposite_alive_account}, append_opposite_zero_lamport_account: {append_opposite_zero_lamport_account}, normal_account_count: {normal_account_count}");
                                 let db = AccountsDb::new_single_for_tests();
                                 let slot5 = 5;
+                                // don't do special zero lamport account handling
+                                db.set_latest_full_snapshot_slot(0);
```

Comment:
because we do unwrap_or(true) related to latest_full_snapshot_slot, this has to be a Some value that is older than the slots we want to work on in the test.